installers/install.ps1

```diff
@@ -98,54 +96,19 @@ function hasWritePermission([string] $path)
     # $acl = Get-Acl $path -ErrorAction 'silentlycontinue'
     # return (($acl.Access | Select-Object -ExpandProperty IdentityReference) -contains $user)
     $thefile = "activestate-perms"
-    New-Item -Path (Join-Path $path $thefile) -ItemType File -ErrorAction 'silentlycontinue'
-    $occurance = errorOccured $True
+    New-Item -Path (Join-Path $path $thefile) -ItemType File -ErrorAction 'SilentlyContinue' -ErrorVariable err
```

Comment:
Does the casing matter here? This occurrence is not the same as others in the diff where it is `silentlycontinue`.

Reply:
It is all case insensitive. Windows!